src/url.rs

```diff
@@ -168,6 +199,23 @@ impl Url {
             url.add_child(news.to_xml()?)?;
         }
 
+        // add <xhtml:link>, if any exist
+        if let Some(alternates) = self.alternates {
+            for alternate in alternates {
+                let mut alternate_link: XMLElement = XMLElement::new("xhtml:link");
+                alternate_link.add_attribute("rel", "alternate");
+                alternate_link.add_attribute("hreflang", &alternate.hreflang);
+                alternate_link.add_attribute("href", &alternate.href);
+                url.add_child(alternate_link)?;
+            }
+        }
+        if let Some(canonical) = self.canonical {
+            let mut canonical_link: XMLElement = XMLElement::new("xhtml:link");
+            canonical_link.add_attribute("rel", "canonical");
```

Comment:
Can you link the official Google documentation for "canonical" `xhtml:link`? I can't find it for some reason.

Reply:
Is this different than `hreflang="x-default"`? 
Context: https://developers.google.com/search/docs/specialty/international/localized-versions#xdefault

Reply:
I have removed this.